tests/cli/test_console.py

```diff
@@ -806,7 +831,7 @@ def test_json_properties_get_table_does_not_exist(catalog: InMemoryCatalog) -> N
     runner = CliRunner()
     result = runner.invoke(run, ["--output=json", "properties", "get", "table", "doesnotexist"])
     assert result.exit_code == 1
-    assert result.output == """{"type": "NoSuchTableError", "message": "Table does not exist: ('doesnotexist',)"}\n"""
+    assert result.output == """{"type": "ValueError", "message": "Empty namespace identifier"}\n"""
```

Comment:
nit: shouldnt this be `NoSuchTableError`? maybe the namespace needs to be created first